lib/team.rb

```diff
@@ -0,0 +1,28 @@
+class Team
+  attr_reader :country,
+              :players
+
+  def initialize(country)
+    @country = country
+    @players = []
+    @eliminated = false
+  end
+
+  def eliminated?
+    @eliminated
+  end
+
+  def eliminate_team
```

Comment:
According to the spec, we should be able to say `team.eliminated = true`. This allows us to call `team.eliminate_team`, so it isn't exactly correct. What you want to do here is define the method `def eliminate=(new_value)`, or make an attr_writer